src/shadowbox/docker/cmd.sh

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export SB_PUBLIC_IP=${SB_PUBLIC_IP:-$(curl https://ipinfo.io/ip)}
```

Comment:
Let's keep this, otherwise those not setting SB_PUBLIC_IP or the config will break